src/Microsoft.Sbom.Api/SBOMGenerator.cs

```diff
@@ -27,22 +28,24 @@ namespace Microsoft.Sbom.Api
     /// </summary>
     public class SBOMGenerator : ISBOMGenerator
     {
-        private readonly ApiConfigurationBuilder configurationBuilder;
-        private readonly StandardKernel kernel;
-        private readonly IFileSystemUtils fileSystemUtils;
-
-        public SBOMGenerator()
+        private readonly IWorkflow<SBOMGenerationWorkflow> generationWorkflow;
+        private readonly ManifestGeneratorProvider generatorProvider;
+        private readonly IRecorder recorder;
+        private readonly IEnumerable<ConfigValidator> configValidators;
+        private readonly ConfigSanitizer configSanitizer;
+
+        public SBOMGenerator(
```

Comment:
Since we're changing API anyway, maybe we can also fix names here to follow conventions? `SBOMGenerator` to `SbomGenerator` and `GenerateSBOMAsync` to `GenerateSbomAsync`. I guess that's only reasonable if we expect breaking change for customers

Reply:
It would also make sense to do it in separate PR, if we're willing to do that, but don't want to overcomplicate current PR